index.html

```diff
@@ -337,6 +356,17 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                                        <span>Member, Embedded Team</span>
                                     </div>
                                  </div>
+                              </article>
+
```

Comment:
Did you add Shivang's information also along with your commit? He already did that.